guides/source/layouts_and_rendering.md

```diff
@@ -1210,7 +1210,7 @@ Partials are very useful in rendering collections. When you pass a collection to
 
 When a partial is called with a pluralized collection, then the individual instances of the partial have access to the member of the collection being rendered via a variable named after the partial. In this case, the partial is `_product`, and within the `_product` partial, you can refer to `product` to get the instance that is being rendered.
 
-There is also a shorthand for this. Assuming `@products` is a collection of `product` instances, you can simply write this in the `index.html.erb` to produce the same result:
```

Comment:
Looks like that this `product` is not a class name, but a local variable in the partial, isn't it?

Reply:
It seems it is saying that the instances inside the collection are of type class `Product`, so it would be ok to capitalize it.
what do you think?

Reply:
@kamipo The previous paragraph does, but the corrected sentence is something entirely different. It is explicitly stating, and correctly stating, that `@products` is a collection of `Product` instances. There is no such thing as a `product` instance, as we only have instances of classes. `product` is not a class name, but `Product` is.